### PR TITLE
[FEATURE] Bank-transaction-import

### PR DESCRIPTION
**Pull request by Civolilah**
Here is implemented solution for bank transaction import as all imports in react app.

Here are screenshots of how it looks:

image

image

transactionImport

### CODE REVIEW

**Comment by Civolilah** [comment]
@turbo124 Please give me details how to complete the execution of this task as you mentioned on Jira's user story.

**Comment by turbo124** [comment]
@beganovich this looks good to me from function perspective :)

File: src/components/import/UploadImport.tsx [hunk 1]
```diff
@@ -167,6 +169,32 @@ export function UploadImport(props: Props) {
                 </Tr>
               )
             )}
+            {props.entity === 'bank_transaction' && (
+              <Tr>
+                <Td className="space-x-2">
+                  <span>Bank account</span>
```
Comment (beganovich) [inline src/components/import/UploadImport.tsx:175]: Missing translation.

**Review by beganovich — CHANGES_REQUESTED** [review]
Looks great, just add a missing translation.

File: src/components/import/UploadImport.tsx (hunk 1 above)
Comment (Civolilah) [reply src/components/import/UploadImport.tsx:175]: > Missing translation.

Great catch, fixed.

**Comment by beganovich** [comment]
Looks good, thanks :+1:

**Pull request merged**